sklearn/utils/__init__.py

```diff
@@ -565,7 +565,7 @@ def resample(*arrays,
         return resampled_arrays
 
 
-def shuffle(*arrays, **options):
+def shuffle(*arrays, random_state=None, n_samples=None):
```

Comment:
Thanks! Did you intentionally include this in this PR?

Reply:
No, was trying to figure out how to fix that.  That was from a PR earlier today.

Reply:
Not intentionally ... It was supposed to be a second PR.
We are going to fix it.

Reply:
Let me know if you need help!